context/ChainsContext/helpers.tsx

```diff
@@ -26,7 +26,7 @@ export const isChainInfoFilled = (chain: Partial<ChainInfo>): chain is ChainInfo
       chain.nodeAddresses?.length &&
       chain.denom &&
       chain.displayDenom &&
-      chain.displayDenomExponent &&
+      chain.displayDenomExponent !== undefined &&
```

Comment:
```suggestion
      typeof chain.displayDenomExponent === "number" &&
```

I guess this is a bit safer and clearer here. Otherwise null would pass the check:

```
> var a = null
undefined
> a >= 0
true
```

Reply:
Maybe we can just use `Number.isSafeInteger`?

Reply:
Sure, that is probably stricter and kicks out things like positive infinity